test/e2e/resources/replicationgroup_cme_ngc.yaml

```diff
@@ -0,0 +1,24 @@
+# CME replication group. "NGC" means that the field nodeGroupConfiguration is specified, with a detailed configuration
+# for each shard
+apiVersion: elasticache.services.k8s.aws/v1alpha1
+kind: ReplicationGroup
+metadata:
+  name: $RG_ID
+spec:
+  cacheNodeType: cache.t3.micro
+  engine: redis
+  nodeGroupConfiguration:
+    - nodeGroupID: $NGID1
+      primaryAvailabilityZone: us-west-2a
+      replicaAvailabilityZones:
+        - us-west-2b
+        - us-west-2c
+      replicaCount: 2
+    - nodeGroupID: $NGID2
+      primaryAvailabilityZone: us-west-2b
+      replicaAvailabilityZones:
+        - us-west-2c
+        - us-west-2a
```

Comment:
Just for my own information, if `replicaCount` is `2`, and the `replicaAvailabilityZones` contains two AZs that are *not* the `primaryAvailabilityZone`, does that mean that there will be a total of 2 instances (the primary in the primary AZ and one replica in one of the replica AZs)? Or does that mean there will be a total of 3 instances (the primary in the primary AZ and one replica in each of the replica AZs)?

Reply:
This should produce a total of 3 instances in the node group – 1 primary and 2 replicas